### PR TITLE
doc: add meeting notes for Feb 14 2018

### PR DESCRIPTION
**Pull request by MylesBorins**
@nodejs/modules should we wait until the next meeting to approve / land this?

### CODE REVIEW

**Comment by MylesBorins** [comment]
I've locked the meeting notes doc to changes so please comment in here for any necessary change. It looks like we are missing people in the attendance. If you were there and not included please comment and I'll add you.

File: doc/meetings/2017-02-14.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=QURheOVmqDk
+* **GitHub Issue**: https://github.com/nodejs/modules/issues/25
+* **Minutes Google Doc**: https://docs.google.com/document/d/17T8ZeF2mUPffiDvulqd_HqD_JrTNaf8ch2dFT1hMfD0/edit?usp=sharing
+
+## Present
+
+Benjamin Gruenbaum (@benjamingr)
```
Comment (devsnek) [inline doc/meetings/2017-02-14.md]: these all need two spaces after them

Comment (MylesBorins) [reply doc/meetings/2017-02-14.md]: could you push a commit that fixes that?

File: doc/meetings/2017-02-14.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Zack Schuster (@zackschuster)
+
+## Agenda
+
+### Announcements
+
+### nodejs/modules
+
+* How often and when should we meet? [#2](https://github.com/nodejs/modules/issues/2)
+  - Meeting doodle - https://doodle.com/poll/vdb8cgz483zzt2t - Default is in EST 4PM
```
Comment (sebdeckers) [inline doc/meetings/2017-02-14.md]: @MylesBorins This shows 404 for me. Was it the doodle for this call, or for future calls?

File: doc/meetings/2017-02-14.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+# Node.js Foundation Modules Team Meeting 2018-02-14
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=QURheOVmqDk
+* **GitHub Issue**: https://github.com/nodejs/modules/issues/25
+* **Minutes Google Doc**: https://docs.google.com/document/d/17T8ZeF2mUPffiDvulqd_HqD_JrTNaf8ch2dFT1hMfD0/edit?usp=sharing
+
+## Present
```
Comment (benjamingr) [inline doc/meetings/2017-02-14.md]: Not a big deal, but for some of those the nicks are capitalized (like DanielRosenwasser) and for some are not (like fishrock123).

File: doc/meetings/2017-02-14.md (hunk 2 above)
Comment (MylesBorins) [reply doc/meetings/2017-02-14.md]: fixed

File: doc/meetings/2017-02-14.md (hunk 3 above)
Comment (MylesBorins) [reply doc/meetings/2017-02-14.md]: I think this is fine

**Pull request merged**